### DISCUSSION TITLE
AWX, Tower, AAP integration

### DISCUSSION THREAD

**Discussion by WimpyMan** [ansibleguy76/ansibleforms#130] [Ideas]
Hi,
Have you looked at what it would take to integrate it to awx? It would be so awesome to be able to replace the survey with those forms!

Cheers!

**Comment by ansibleguy76** [comment]
that is what it is built for. you can make a connection with awx.

**Comment by ansibleguy76** [comment]
In settings you can make a connection to AWX / Tower / AAP.  It is a rest integration.  It is NOT embedded into AWX.
